### DISCUSSION TITLE
Are the docs for "Scroll to the first unread" ahead of the feature?

### DISCUSSION THREAD

**Discussion by hanna-becker** [flyerhq/flutter_chat_ui#320] [Q&A] [answered]
Today I tried implementing scrolling to the first unread message when opening the chat, as outlined here: https://docs.flyer.chat/flutter/chat-ui/advanced-usage#scroll-to-the-first-unread

I tried both versions, the one with and the one without pagination. I actually need the one with pagination.

However, in the latest release (1.6.4) the classes ScrollToUnreadOptions and ChatState are not exposed in the library. They are exposed when I instead use an unreleased version (current state of the main branch). Seems like the docs are ahead of the release here? Still, neither of the examples work in that unreleased version, and no scrolling happens when I open the chat. I do see the unread message header in the correct position, though, after manually scrolling there.

I guess I cannot expect things to work on a random unreleased version, but what confuses me is that it's already in the docs. And also the fact that the docs say they reference version "Chat UI 1.6.0" in the upper left corner, and I wonder if that's true at all, and if so, whether I can change it to see docs for the latest released (1.6.4).

**Comment by demchenkoalex** [comment, marked as answer]
I think I saw the same issue and replied there. All has been released since. Sorry for the confusion :)
